ignite/contrib/handlers/time_profilers.py

```diff
@@ -3,6 +3,7 @@
 from typing import Any, Callable, Dict, Iterable, List, Mapping, Sequence, Union
 
 import torch
+from torch._C import dtype
```

Comment:
This is unnecessary, please remove this import :)

Reply:
😄 Result of PyLance in VSCode. I think I will get back to old one.